README.md

```diff
@@ -1,7 +1,17 @@
 # data-ingestion-lib
-Data ingestion lib is library to use Azure Storage files with  camel integration and save Azure files in local project DB
+Data ingestion lib creates reusable framework library with Spring boot and Apache Camel Integration to read CSV files from Azure Blob
+and transform/converts and stores it in spring microservice project database.
+
+This library has been published in bin tray with git push actions on new release (https://bintray.com/hmcts/hmcts-maven/data-ingestion-lib).
+
+# To build the project in local execute the following command
+./gradlew build 
+
+# How to use library
+Common library properties like email settings configured in library and customized properties should be configured with specific 
+microservices eg. rd-judicial-data-load (https://github.com/hmcts/rd-judicial-data-load)
+
+
```

Comment:
Its good that you have updated readme for reference but you can also mention the build.gradle entry for sample. Also will it be possible to mention latest version in readme file, so that user can directly see which latest version to use.

Reply:
👍
you should include the gradle entry in read me so that anyone who refers know how to include this in their project

Reply:
readme file could provide more details on what this library does and how it can be used for data load
Add instructions to what a team/user needs to do for data load after adding library to project